_docs/getting-started/mac.md

```diff
@@ -6,22 +6,44 @@ The Mac build works easiest with Anaconda. Always pull the latest from github, s
 
 ## Anaconda Install Path
 
-[Anaconda](https://www.continuum.io/downloads). Python 2.7 version is needed for Caffe2, and Anaconda is recommended. Skip this section to find brew/pip install directions if you are not using Anaconda.
+[Anaconda](https://www.continuum.io/downloads). Python 2.7 version is needed for Caffe2, and Anaconda is recommended. Skip this section to find brew/pip install directions if you are not using Anaconda. The following commands will install caffe2 similarly to how `conda install caffe2` would.
 
 ```bash
 git clone --recursive https://github.com/caffe2/caffe2.git && cd caffe2
 conda build conda
 conda install caffe2 --use-local
 ```
 
-This will build caffe2 using [conda build](https://conda.io/docs/user-guide/tasks/build-packages/recipe.html), with the flags specified in `conda/build.sh` and the packages specified in `conda/meta.yaml`. `conda build` will create a conda package (tarball) on your machine, which `conda install` then installs. To build caffe2 with different settings, change the dependencies in `meta.yaml` and the `CMAKE_ARGS` flags in `conda/build.sh` and run the above command again. 
+This will build caffe2 using [conda build](https://conda.io/docs/user-guide/tasks/build-packages/recipe.html), with the flags specified in `conda/build.sh` and the packages specified in `conda/meta.yaml`. `conda build` will create a conda package (tarball) on your machine, which `conda install` then installs. To build caffe2 with different settings, change the dependencies in `meta.yaml` and the `CMAKE_ARGS` flags in `conda/build.sh` and run the above commands again. 
 
 If your default Anaconda Python is not 2.7, you can install a different version of Python using `conda create --name python2 python=2` (`python2` can be any name you like.)  Subsequently, if you `source activate python2`, your path will be adjusted so that you get `python2`. See [this page](https://conda.io/docs/user-guide/tasks/manage-environments.html) on managing conda environments for more info.
 
 ### Optional GPU Support
 
 In the instance that you have a NVIDIA supported GPU in your Mac, then you should visit the NVIDIA website for [CUDA](https://developer.nvidia.com/cuda-downloads) and [cuDNN](https://developer.nvidia.com/cudnn) and install the provided binaries. You will have to remove `-DUSE_CUDA=OFF` and `-DUSE_NCCL=OFF` flags from `conda/build.sh`.
 
+### Custom Anaconda Install
+
+If you plan to change the source code of caffe2 or want to customize the build process, you may want to bypass conda and call cmake manually. The following commands will build caffe2 in a directory called `build` under your caffe2 root and install caffe2 in a conda env. In this example Anaconda is installed in `~/anaconda2`.
```

Comment:
does this mean that conda install caffe2 won't pick up local changes? I was under an impression that it would

Reply:
If you `conda build` with your changes and then `conda install --use-local` it will. I thought this might be a slow process for people who want to change the source a lot. I'll change this to 'plan to change the source code a lot' or something like that